api/hpa.go

```diff
@@ -118,11 +118,6 @@ func DeleteHpaResource(c *gin.Context) {
 func GetHpaResource(c *gin.Context) {
 	scaleTarget, ok := ginutils.RequiredQuery(c, "scaleTarget")
 	if !ok {
-		c.JSON(http.StatusBadRequest, pkgCommmon.ErrorResponse{
```

Comment:
Why this block got deleted?

Reply:
Sorry I missed that, probably it should have a name like `RequiredQueryOrError` just to make it more understandable.

Reply:
You're right. I will rename this function

Reply:
OrAbort is probably more Gin-ish! :)